src/ruby/lib/grpc/generic/service.rb

```diff
@@ -31,6 +31,7 @@ module GenericService
     #
     # @param s [String] the string to be converted.
     def self.underscore(s)
+      s = +s
```

Comment:
can we add a comment for motivation, and how this is for upcoming ruby 2.7 compatibility?

Reply:
Sure.